astropy/io/fits/tests/test_header.py

```diff
@@ -280,7 +280,7 @@ def __getitem__(self, key):
 
         header.update(NAXIS1=100, NAXIS2=100)
         assert set(header.keys()) == {"FOO", "A", "B", "HELLO", "NAXIS1", "NAXIS2"}
-        assert set(header.values()) == {"BAR", 1, 2, 100, 100}
+        assert set(header.values()) == {"BAR", 1, 2, 100}
```

Comment:
I am wondering if there should be a better assertion here to verify that "100" is returned twice by ``header.values()``. But there is no in-built way of asserting that and would require explicit code. If such verification is not required, above change should be good.

Reply:
We can do tuple comparison. Python dicts are now ordered